python/Ganga/test/GPI/GangaList/TestCopy.py

```diff
@@ -27,18 +25,18 @@ def testCopy(self):
 
         gl = GangaList()
 
-        numberOfFiles = 5
+        numberOfFiles = 100
 
         for _ in range(numberOfFiles):
             # add something which is generally not allowed by GangaList
-            gl.append(self._makeRandomTFile())
+            gl.append([self._makeRandomTFile()])
```

Comment:
doesn't this equate to:
```
[ [TFile], [TFile], [TFile], ....]
```
instead of
```
[ TFile, TFile, TFile, ...]
```

That may be what's needed (e.g. to test nested lists) but isn't that in another test?